### PR TITLE
Add JSON_RPC_API.md with transaction apis

### PR DESCRIPTION
**Pull request by platfowner**

### CODE REVIEW

**Review by minsulee2 — APPROVED** [review]
LGTM!

File: JSON_RPC_API.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+`Array` - A list of pending transactions.
+
+**Example**
+
+Request
+```
+curl -X POST --header 'Content-Type: application/json' --data '{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "ain_getPendingTransactions",
```
Comment (jiyoung-an) [inline JSON_RPC_API.md:24]: where is this function defined? (is blockchain?)

**Review by jiyoung-an — APPROVED** [review]
LGTM with a question. Thanks!

File: JSON_RPC_API.md (hunk 1 above)
Comment (platfowner) [reply JSON_RPC_API.md:24]: It's defined in json_rpc/transaction.js:
https://github.com/ainblockchain/ain-blockchain/blob/develop/json_rpc/transaction.js#L64

**Comment by platfowner** [comment]
Thanks for the review! @minsulee2 @jiyoung-an Now merging..

**Pull request merged**